.github/workflows/release.yml

```diff
@@ -270,118 +267,111 @@ jobs:
   #         CONFLUENCE_USERNAME: ${{ secrets.VBR_CONFLUENCE_USER }}
   #         CONFLUENCE_PASSWORD: ${{ secrets.VBR_CONFLUENCE_PASSWORD }}
 
-    #  - name: Create Jira Release
-    #    shell: bash
-    #    run: ./packages/Be.Vlaanderen.Basisregisters.Build.Pipeline/Content/ci-jira.sh
-    #    env:
-    #      CONFLUENCE_TITLE: ${{ env.REPOSITORY_NAME }}
-    #      CONFLUENCE_USERNAME: ${{ secrets.VBR_CONFLUENCE_USER }}
-    #      CONFLUENCE_PASSWORD: ${{ secrets.VBR_CONFLUENCE_PASSWORD }}
-    #      JIRA_PREFIX: Building
-    #      JIRA_PROJECT: GAWR
-    #      JIRA_VERSION: ${{ needs.release.outputs.version }}
-
   push_images:
     if: needs.release.outputs.version != 'none'
     needs: [ release ]
     name: Push images
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        image: [ 'integration-veka', 'suspicious-cases-api' ]
```

Comment:
why removing the matrix?

Reply:
Based upon the release of ticketing-system where these steps are duplicated https://github.com/Informatievlaanderen/ticketing-system/blob/main/.github/workflows/release.yml#L309-L337. I'll use the matrix here and remove the duplication.

Reply:
Done